test/e2e/utils/config.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2018-2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"flag"
+	"io/ioutil"
+	"regexp"
+
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
+
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	configContent = flag.String("nfd.e2e-config", "", "Configuration parameters for end-to-end tests")
+	E2EConfigFile *e2eConfig
```

Comment:
I think I get what you mean, but we will require a helper getter because `test/e2e/node_feature_discovery.go` wants to access this data. I'll try something.